io-ballerina/open.bal

```diff
@@ -16,6 +16,15 @@
 
 import ballerina/jballerina.java;
 
+# Represents a file opening options for writing.
+#
+# + OVERWRITE - Overriwrite(truncate the existing content)
+# + APPEND - Append to the existing content
+public enum FileOpenOption {
```

Comment:
Does it better if we name this enum as `FileWriteOption` Since we only use this in the write operations

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/68/commits/a93c3dd4b4b049aca260f34d1187b4fd90893f1b